src/reducers/archiveThought.js

```diff
@@ -85,16 +95,24 @@ const archiveThought = (state, { path }) => {
     }
   }
 
+  /** Gets the next sibling context in the context view. */
+  const nextContext = () => {
+    const contextsSortedAndRanked = getContextsSortedAndRanked(state, headValue(contextOf(path)))
+    const contextsFiltered = contextsSortedAndRanked
+      .filter(({ context }) => head(context) !== '=archive')
+      .map((context, i) => Object.assign({}, context, { rank: i }))
+    return contextsFiltered.find(c => head(c.context) !== head(context))
```

Comment:
Rather than doing `filter` and `map` first, combine the logic into `find`. This saves unnecessary iterations when the matching context is at the beginning of the list.